File: C Programming/Competitive Programming/Coding Challenges 8/program2.c
// 2. Write a program which accept width & height of rectangle from user and calculate its area. (Area = Width * Height)
// Input : 5.3 9.78
// Output : 51.834

#include<stdio.h>

double ReactArea(float RectangleArea1,float RectangleArea2)
{
   double dArea = 0.0;

   dArea = RectangleArea1 * RectangleArea2;

   return dArea;
}
int main()
{
    float fValue1 = 0.0f,fValue2=0.0f;
    double dRet = 0.0;

    printf("Enter width: ");
    scanf("%f",&fValue1);

    printf("Enter height: ");
    scanf("%f",&fValue2);

    dRet = ReactArea(fValue1,fValue2);

    printf("Area is :%.3f",dRet);

    return 0;
}

File: Java Programming/Competitive Programming/Coding Challenges 14/program2.java
// 2. Write a program which accept width & height of rectangle from user and calculate its area. (Area = Width * Height)
// Input : 5.3 9.78
// Output : 51.834

import java.util.Scanner;

class program2
{
    static double ReactArea(double[] dValue)
    {
        double dArea = dValue[0] * dValue[1];

        return dArea;
    }

    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        double[] dValue = new double[2];

        System.out.print("Enter width: ");
        dValue[0] = sobj.nextDouble();

        System.out.print("Enter height: ");
        dValue[1] = sobj.nextDouble();

        double dRet = ReactArea(dValue);

        System.out.printf("Area is : %.3f", dRet);

        sobj.close();
    }
}